pages/plugins/using.md

```diff
@@ -115,6 +115,24 @@ steps:
 
 ```
 
+You can override an anchor by using `<<:` before the _alias_. This allows overriding parts of the anchor without needing to repeat the all plugin configuration or creating multiple anchors.
```

Comment:
Thanks for your contribution @tomowatt ! Looks great!

Out of interest, what is the "all plugin configuration"?

Reply:
Apologies @gilesgas originally had a different example which was contextual to that text, but correct it now.